src/options.js

```diff
@@ -268,6 +268,16 @@ exports.bool = {
     singleGroups: false,
 
     /**
+     * When set to true, the use of VariableStatements are forbidden.
+     * For example:
```

Comment:
Although not all the option descriptions include this, I think it's helpful if the explanation also has some text explaining the motivation for the feature. For example:

> `let` and `const` statements are block scoped, so they offer more precise control over the availability of a binding and more informative static errors. The function scoping of `var` statements enables usage patterns that are difficult to read and error-prone.

...but I'd be happy with whatever text you'd like to include.